File-Formats/DocIO/Create-Word-document-in-WinUI.md

```diff
@@ -18,19 +18,19 @@ To use the WinUI 3 project templates, install the Windows App SDK extension for
 
 Step 1: Create a new C# WinUI Desktop app. Select Blank App, Packaged with WAP (WinUI 3 in Desktop) from the template and click the **Next** button.
 
-![Create the WinUI Desktop app in Visual Studio](WinUI_Images/Create_Desktop_Project.png)
+![Create the WinUI Desktop app in Visual Studio](winui_images/create-winui-desktop-project.png)
 
 Step 2: Enter the project name and click **Create**.
 
-![Create a project name for your new project](WinUI_Images/Desktop_Configure.png)
+![Create a project name for your new desktop project](winui_images/create-desktop-project-name-in-winui-word.png)
 
 Step 3: Set the Target version to Windows 10, version 2004 (build 19041) and the Minimum version to Windows 10, version 1809 (build 17763) and then click **OK**.
 
-![Set the target version](WinUI_Images/Target_Version.png)
+![Set the target version for desktop](winui_images/target-version-in-winui-word.png)
 
 Step 4: Install the Syncfusion.DocIO.NET NuGet package as a reference to your project from the [NuGet.org](https://www.nuget.org/).
 
-![Install the DocIO .NET NuGet package](WinUI_Images/Install_Nuget.png)
+![Install the Desktop DocIO .NET NuGet package](winui_images/install--nuget-in-winui-word.png)
```

Comment:
double hyphen is there in image name. check and correct it.

Reply:
Removed double hyphen